samza-core/src/main/java/org/apache/samza/config/TaskConfig.java

```diff
@@ -105,6 +105,7 @@
   private static final String BROADCAST_STREAM_PATTERN = "^[\\d]+$";
   private static final String BROADCAST_STREAM_RANGE_PATTERN = "^\\[[\\d]+\\-[\\d]+\\]$";
   public static final String CHECKPOINT_MANAGER_FACTORY = "task.checkpoint.factory";
+  public static final String INTERNAL_CHECKPOINT_MANAGER_CONSUMER_STOP_AFTER_FIRST_READ = "samza.internal.task.checkpoint.consumer.stop.after.first.read";
```

Comment:
Lets maybe add a one-line description of what this config does?
Also if it'd be possible to shrink the name somewhat.

Reply:
I initially thought of something like `samza.internal.task.checkpoint.consumer.keep.open` but thought that the name in the PR currently is more expressive of behavior.

Thoughts?

Reply:
sure lets go with that one then